### PR TITLE
Changed build.gradle to accept higher Spock versions.

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -56,7 +56,7 @@ subprojects {
 
 	dependencies {
 		compile localGroovy()
-		testCompile('org.spockframework:spock-core:1.0-groovy-2.3') {
+		testCompile('org.spockframework:spock-core:1.0-groovy-2.3+') {
```
Comment (szpak) [inline build.gradle:59]: Wouldn't it catch `1.0-groovy-2.4`?

**Comment by OlgaMaciaszek** [comment]
Would that be a bad thing?

**Comment by szpak** [comment]
For using with Groovy 2.3 (which is provided by Gradle) - yes. It would fail in runtime with the very meaningful error message :).

**Comment by OlgaMaciaszek** [comment]
:) Let's leave it as it is then.

**Pull request closed without merging**